biorbd_optim/penalty.py

```diff
@@ -397,3 +426,4 @@ class PenaltyType(Enum):
     ALIGN_SEGMENT_WITH_CUSTOM_RT = PenaltyFunctionAbstract.Functions.align_segment_with_custom_rt
     ALIGN_MARKER_WITH_SEGMENT_AXIS = PenaltyFunctionAbstract.Functions.align_marker_with_segment_axis
     CUSTOM = PenaltyFunctionAbstract.Functions.custom
+    TRACK_CONTACT_FORCES = PenaltyFunctionAbstract.Functions.minimize_contact_forces
```

Comment:
Please move this right after TRACK_ALL_CONTROLS

Reply:
done